spec/abstractops.html

```diff
@@ -199,8 +201,41 @@ <h1>ToSmallestTemporalDurationUnit ( _normalizedOptions_, _fallback_, _disallowe
     </emu-alg>
   </emu-clause>
 
-  <emu-clause id="sec-temporal-validatetemporaldifferenceunits" aoid="ValidateTemporalDifferenceUnits">
-    <h1>ValidateTemporalDifferenceUnits ( _largestUnit_, _smallestUnit_ )</h1>
+  <emu-clause id="sec-temporal-torelativetemporalobject" aoid="ToRelativeTemporalObject">
+    <h1>ToRelativeTemporalObject ( _options_ )</h1>
+    <p>
+      The abstract operation ToRelativeTemporalObject examines the value of the `relativeTo` property of its _options_ argument, and attempts to return a Temporal.ZonedDateTime instance <mark>(TODO)</mark>, Temporal.DateTime instance, or *undefined*, in order of preference.
+      If none of those are possible, the operation throws a *RangeError*.
+    </p>
+    <emu-alg>
+      1. Assert: Type(_options_) is Object.
+      1. Let _value_ be ? Get(_options_, _property_).
+      1. If _value_ is *undefined*, then
+        1. Return _value_.
+      1. If Type(_value_) is Object, then
+        1. If _value_ has an [[InitializedTemporalDateTime]] internal slot, then
+          1. Return _value_.
+        1. Let _calendar_ be ? Get(_value_, *"calendar"*).
+        1. If _calendar_ is *undefined*, then
+          1. Set _calendar_ to ! GetDefaultCalendar().
+        1. Set _calendar_ to ? ToTemporalCalendar(_calendar_).
+        1. Let _fields_ be ? ToTemporalDateTimeRecord(_value_).
+        1. Let _dateFromFields_ be ? Get(_calendar_, *"dateFromFields"*).
+        1. Let _dateOptions_ be ! OrdinaryObjectCreate(%Object.prototype%).
+        1. Let _date_ be ? Call(_dateFromFields_, _calendar_, « _fields_, _dateOptions_, %Temporal.Date% »).
+        1. Return ? CreateTemporalDateTime(_date_.[[ISOYear]], _date_.[[ISOMonth]], _date_.[[ISODay]], _fields_.[[Hour]], _fields_.[[Minute]], _fields_.[[Second]], _fields_.[[Millisecond]], _fields_.[[Microsecond]], _fields_.[[Nanosecond]], _calendar_).
```

Comment:
I think it would be only a minimal savings, used in Date.prototype.toDateTime and Time.prototype.toDateTime (but then why would we prioritize that over TemporalTimeToDateTime)?